conf/nginx/nginx.conf.web.https.default.template

```diff
@@ -153,7 +161,7 @@ server
         # End stray redirect hack
 
         # Proxy to Zimbra Webclient Upstream
-        proxy_pass       ${web.upstream.webclient.target};
+        proxy_pass       $login_upstream;
```

Comment:
when web.login.upstream.disable is true, should it go to proper webclient target instead of empty login target?

Reply:
@silentsakky This is indeed broken with web.login.upstream.disable=true (zimbra_login target undefined), we reverted to zimbra_webclient target to make it work again while upgrading to Zimbra 10.